### PR TITLE
WIP: check both dev and stable dart, use container runtime

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: dart
-
-dart: dev
-
+sudo: false
```
Comment (devoncarew) [inline .travis.yml]: We do use `sudo` in this script -

**Comment by devoncarew** [comment]
We use `sudo` to install chromedriver, in order to run the webdriver tests.

**Comment by kevmoo** [comment]
Other than the `mv` in `.travis.yml`, where is this needed?

**Comment by devoncarew** [comment]
Hmm, just there. I suppose you could work around this by adding the chromedriver install location to the path.

**Comment by kevmoo** [comment]
Exactly! :smile:

**Comment by DrMarcII** [comment]
I went ahead and incorporated this change into in #66

**Pull request closed without merging**